### DISCUSSION TITLE
Is it possible to automatically update playlist from RSS feed?

### DISCUSSION THREAD

**Discussion by houseofservers** [AzuraCast/AzuraCast#5812] [Q&A] [answered]
Never tried this but is it possible that the playlist is updated based on the RSS feed.

I have received an rssFeed containing audio track and want only the newest ones to be played.

Kindly let me know if it's possible.

I am a bit new to this so a step by step procedure on how to do it would be helpful

**Comment by BusterNeece** [comment, marked as answer]
@Hadhab Not currently, no, but you can always request it as a new feature: https://features.azuracast.com/
